main.js

```diff
@@ -300,20 +302,18 @@ function setVideoElements() {
 
   const video = Videos.list[Videos.video];
 
-  document.getElementsByTagName("source")[0].src = "video/" + video.file;
+  document.getElementsByTagName("source")[0].src = "/video/" + video.file;
```

Comment:
Good point. I hadn't thought about it being installed somewhere other than at the top.